src/functional/ffront/past_to_itir.py

```diff
@@ -107,7 +107,14 @@ def visit_Program(
         #  a workaround we add additional arguments to the fencil definition
         #  containing the size of all fields. The caller of a program is (e.g.
         #  program decorator) is required to pass these arguments.
-        size_params = self._gen_size_params_from_program(node)
+
+        params = [itir.Sym(id=inp.id) for inp in node.params]
```

Comment:
It's not your code, but can you try if 
```suggestion
        params = self.visit(node.params)
```
works? I expect that all node.params are past.Name, but not sure if I overlook something.

Reply:
@tehrengruber self.visit does not work here